runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -456,9 +455,7 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 		int groupId = widgetLoaded.getGroupId();
 		if (groupId == WidgetID.SLAYER_REWARDS_GROUP_ID)
 		{
-			Widget rewardOptions = client.getWidget(WidgetInfo.SLAYER_REWARDS_REWARD_OPTIONS);
-			Widget doubleTroubleCheckbox = rewardOptions.getChild(REWARD_DOUBLE_TROUBLE_CHECKBOX_ID);
-			doubleTroubleEnabled = (doubleTroubleCheckbox.getSpriteId() == REWARD_ENABLED_SPRITE);
+			doubleTroubleEnabled = (client.getVar(VarPlayer.DOUBLE_TROUBLE) >> 12 & 1) == 1;
```

Comment:
This magic number should be replaced by some definition of what the DOUBLE_TROBULE bits are. (I believe Abex has a file with these definitions, and posted it in Discord)